package-lock.json

```diff
@@ -18,6 +18,7 @@
         "axios": "^0.26.1",
         "emoji-picker-react": "^3.5.1",
         "history": "^5.3.0",
+        "material-ui-dropzone": "^3.5.0",
```

Comment:
it seems the package-lock still has the dependencies of material-ui-dropzone installed, did you run the uninstall command or just delete it from the package.json?